frontend/awx/administration/instances/InstanceDetails.cy.tsx

```diff
@@ -89,4 +103,32 @@ describe('Instance Details', () => {
         cy.get('.pf-v5-c-slider__thumb').should('have.attr', 'aria-disabled', 'true');
       });
   });
+
+  it('edit instance button should be hidden for non-k8s system', () => {
+    cy.intercept('GET', '/api/v2/settings/system*', {
+      IS_K8S: false,
+    }).as('isK8s');
+    cy.mount(<InstanceDetails />);
+    cy.get('[data-cy="edit-instance"]').should('not.exist');
+  });
+
+  it('edit instance button should be shown for k8s system', () => {
+    cy.intercept('GET', '/api/v2/settings/system*', {
+      IS_K8S: true,
+    }).as('isK8s');
+    cy.mount(<InstanceDetails />);
+    cy.get('[data-cy="edit-instance"]').should('be.visible');
+    cy.get('[data-cy="edit-instance"]').should('have.attr', 'aria-disabled', 'false');
+  });
+
+  it('only admin users can edit instance', () => {
+    cy.intercept({ method: 'GET', url: '/api/v2/me' }, { fixture: 'adminUser.json' });
```

Comment:
The tests are showing disabled for edit because it's missing a fixture for the system settings. Replacing this intercept with an intercept for is_k8s should cover this use case.